semantic_conventions/metrics/rpc-metrics.yaml

```diff
@@ -0,0 +1,116 @@
+  - id: attributes.metrics.rpc
+    type: attribute_group
+    brief: "Describes RPC metric attributes."
+    attributes:
+      - ref: rpc.system
+      - ref: rpc.service
+      - ref: rpc.method
+      - ref: network.transport
+      - ref: network.type
+      - ref: server.address
+      - ref: server.port
+      - ref: server.socket.address
+      - ref: server.socket.port
+
+  # RPC Server metrics
+  - id: metric.rpc.server.duration
+    type: metric
+    metric_name: rpc.server.duration
+    brief: >
+      Measures the duration of inbound RPC.
+      **Streaming**: N/A.
+    instrument: histogram
+    unit: "ms"
+    note: |
+      While streaming RPCs may record this metric as start-of-batch
+      to end-of-batch, it's hard to interpret in practice.
+
+  - id: metric.rpc.server.request.size
+    type: metric
+    metric_name: rpc.server.request.size
+    brief: >
+      Measures the size of RPC request messages (uncompressed).
+      **Streaming**: Recorded per message in a streaming batch
+    instrument: histogram
+    unit: "By"
+
+  - id: metric.rpc.server.response.size
+    type: metric
+    metric_name: rpc.server.response.size
+    brief: >
+      Measures the size of RPC response messages (uncompressed).
+      **Streaming**: Recorded per response in a streaming batch
+    instrument: histogram
+    unit: "By"
+
+  - id: metric.rpc.server.requests_per_rpc
+    type: metric
+    metric_name: rpc.server.requests_per_rpc
+    brief: >
+      Measures the number of messages received per RPC. Should be 1 for all non-streaming RPCs.
+      **Streaming**: This metric is required for server and client streaming RPCs
+    instrument: histogram
+    unit: "{count}"
+
+  - id: metric.rpc.server.responses_per_rpc
+    type: metric
+    metric_name: rpc.server.responses_per_rpc
+    brief: >
+      Measures the number of messages sent per RPC. Should be 1 for all non-streaming RPCs.
+      **Streaming**: This metric is required for server and client streaming RPCs
+    instrument: histogram
+    unit: "{count}"
+
+  # RPC Client metrics
+  - id: metric.rpc.client.duration
+    type: metric
+    metric_name: rpc.client.duration
+    brief: >
+      Measures duration of outbound RPC
+
+      **Streaming**: N/A.
+    instrument: histogram
+    unit: "ms"
+    note: |
+      While streaming RPCs may record this metric as start-of-batch
+      to end-of-batch, it's hard to interpret in practice.
+
+  - id: metric.rpc.client.request.size
+    type: metric
+    metric_name: rpc.client.request.size
+    brief: >
+      Measures the size of RPC request messages (uncompressed).
+      **Streaming**: Recorded per message in a streaming batch
+    instrument: histogram
+    unit: "By"
+
+  - id: metric.rpc.client.response.size
+    type: metric
+    metric_name: rpc.client.response.size
+    brief: >
+      Measures the size of RPC response messages (uncompressed).
+      **Streaming**: Recorded per response in a streaming batch
```

Comment:
We had a column for "Streaming" which we don't now. The workaround I used is to add it in the description. Do you think that's enough? I also thought of adding it as a Note, but I find that even more "hidden".

Reply:
I added the "streaming" text for `rpc.server.duration` and `rpc.client.duration` as a Note. Now that we target the build-tools 0.19, notes are also rendered in markdown. Closing this as I think it's good enough.